src/renderers/webgl/WebGLUniforms.js

```diff
@@ -568,14 +568,18 @@ function setValueT1( gl, v, textures ) {
 
 	}
 
-	if ( this.type === gl.SAMPLER_2D_SHADOW && emptyShadowTexture === null ) {
+	let emptyTexture2D = null;
```

Comment:
Is there a reason for the `null` assignment?

Reply:
No, not really.

Reply:
eea6cc81fa2210c1297968b782779d3259268164